primo/opt_model/model_options.py

```diff
@@ -191,8 +203,12 @@ def __init__(self, cluster_mapping=None, **kwargs):
             logging.info("Clustering Data in Opt Model Inputs")
             # Construct campaign candidates
             # Step 1: Perform clustering, Should distance_threshold be a user argument?
-            perform_clustering(wd, distance_threshold=10.0)
-
+            if self.config.cluster_formulation == "Agglomerative":
+                perform_agglomerative_clustering(wd, distance_threshold=10.0)
+            else:
+                perform_louvain_clustering(
+                    wd, distance_threshold=10.0, cluster_threshold=300
```

Comment:
Should we expose distance threshold and cluster threshold as both user options in the config block?

Reply:
I think this is a good idea.

Reply:
Added both to model options